build-resx.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs');
+const path = require('path');
+const {create} = require('xmlbuilder2');
+
+const basePath = 'crowdin';
+const outputFilename = 'docs.resx';
+const filesToProcessExt = '.html.md'
+
+const mappings = {
+    'adguard-mail.com': 'legal-github-docs-mail',
+    'adguard.com': 'legal-github-docs-adguard',
+    'adguard-dns.io': 'legal-github-docs-dns',
+    'adguard-vpn.com': 'legal-github-docs-vpn',
+};
+
+console.log('Starting document processing...');
+
+if (!fs.existsSync(basePath)) {
+    console.log(`Creating output directory: ${basePath}`);
+    fs.mkdirSync(basePath);
+}
+
+for (const [sourceDir, crowdinDir] of Object.entries(mappings)) {
+    console.log(`\nProcessing directory: ${sourceDir} -> ${crowdinDir}`);
+
+    if (!fs.existsSync(sourceDir)) {
+        console.log(`Skipping ${sourceDir} - directory does not exist`);
+        continue;
+    }
+
+    const mdFiles = fs.readdirSync(sourceDir, {recursive: true})
+        .filter(file => file.endsWith(filesToProcessExt));
+
+    console.log(`Found ${mdFiles.length} files to process`);
+
+    let xmlBuilder = create({version: '1.0', encoding: 'utf-8'}).ele('root');
+
+    for (const filePath of mdFiles) {
+        console.log(`Processing file: ${filePath}`);
+        const content = fs.readFileSync(path.join(sourceDir, filePath), 'utf8');
+
+        const key = filePath
+            .replace(filesToProcessExt, '')
+            .replaceAll('/', '.');
+
+        console.log(`Generated key: ${key}`);
+
+        xmlBuilder = xmlBuilder.ele('data').att('name', key)
+            .ele('value').txt(content).up()
```

Comment:
Make sure that it uses CDATA for the item contents, this is pretty important for Crowdin, otherwise all special characters will be escaped there.

Reply:
That's why I'd prefer `xml2js`, it has explicit option for using cdata.

Reply:
Ping here

Reply:
Fixed. I'm not sure that CDATA helps here, without it it's cleaner in crowdin.

Reply:
Ok, CDATA is better :)